index.html

```diff
@@ -30,5 +30,94 @@ <h1 class="text-primary-1 ff-crete line-height-1 ">Hey There. <br> I'm Tomi</h1>
            <li> <a href="#"  class="btn-contact linkedin"> </a> </li> 
         </ul>
     </main>
+    <section class="container work-section grid bg-light">
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_72](https://user-images.githubusercontent.com/69638013/165136595-a89a6f90-191b-45cf-b605-07918c5c1a80.png)  | ![Screenshot_71](https://user-images.githubusercontent.com/69638013/165136553-19e02674-ad87-4b90-b73b-435f4586274b.png)

- [ ] Kindly increase the width of each of the cards so it looks exactly like the Figma design 👍🏿